skills/lark-base/references/lark-base-form-submit.md

```
@@ -0,0 +1,147 @@
+  "单选字段": "选项A",
+  "多选字段": ["选项A", "选项B"],
+  "时间字段": "2026-04-27 14:30:00",
+  "复选框字段": true,
+  "人员字段": [{ "id": "ou_7094d131420c8749632145f08fbf114a" }],
+  "关联字段": [{ "id": "recXXXXXXXXXXXX" }],
+  "地理位置字段": { "lng": 116.397428, "lat": 39.90923 }
+}
+```
+
+> **注意：附件类型字段不要写在 `fields` 里。** `fields` 中不包含附件，附件有独立的填写方式，见下方「attachments（附件上传）」章节。
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)